ExtractTable/FileOperations/__init__.py

```diff
@@ -120,8 +120,11 @@ def download_file(self, url: str):
         with requests.get(url, stream=True) as r:
             r.raise_for_status()
             _, r_ext = r.headers['Content-Type'].rsplit('/', 1)
-            fname, f_ext = os.path.basename(url).rsplit('.', 1)
-            ext = r_ext if r_ext else f_ext
+            fname, f_ext = os.path.basename(
+                # >>> 'string'.split(None)[0] == 'string'
```

Comment:
@marktgraham  - Are these comments meant to stay here? Just checking

Reply:
@Nawar-Obidou - yes, I left them there to guide the reader who might wonder what would happen if the kwarg is not there, in which case the string is split about `None`. I wasn't sure, and I was pleasantly surprised that it works with `None`!